src/test/java/uk/ac/ebi/ampt2d/metadata/MetadataApplicationTest.java

```diff
@@ -626,4 +645,35 @@ public void findStudyByTaxonomyName() throws Exception {
                 .andExpect(jsonPath("$..studies.length()").value(0));
     }
 
+    @Test
+    public void findStudyByReleaseDate() throws Exception {
```

Comment:
It seems that my comment was lost. Please also include tests to check that the user receives a proper error when the inputs are wrong.

Reply:
Added.